src/main/scala/com/sumologic/sumobot/core/Receptionist.scala

```diff
@@ -110,49 +108,53 @@ class Receptionist(rtmClient: SlackRtmClient,
       pendingIMSessionsByUserId = pendingIMSessionsByUserId + (userId -> (doneRecipient, doneMessage))
 
     case message: Message if !tooOld(message.ts, message) =>
-      translateAndDispatch(message.channel, message.user, message.text, message.ts)
+      translateAndDispatch(message.channel, message.user, message.text, message.ts, parentTimeStamp = message.thread_ts)
```

Comment:
Let's call this variable `threadTimestamp` to be consistent with what the API calls it, and make the spelling of `Timestamp` consistent with `idTimestamp` (i.e. no upper-case `S`).